fetch-js-deps.sh

```diff
@@ -6,7 +6,7 @@ mkdir -vp resources/public/third-party/pagedown
 mkdir -vp resources/public/third-party/jquery
 
 curl -O http://getbootstrap.com/2.3.2/assets/bootstrap.zip
```

Comment:
```suggestion
curl -O https://getbootstrap.com/2.3.2/assets/bootstrap.zip
```

This download breaks as well without HTTPS